packages/apps/purchase/src/models/sortFields.ts

```diff
@@ -0,0 +1,23 @@
+/*
+ * Axelor Business Solutions
+ *
+ * Copyright (C) 2024 Axelor (<http://axelor.com>).
+ *
+ * This program is free software: you can redistribute it and/or  modify
+ * it under the terms of the GNU Affero General Public License, version 3,
+ * as published by the Free Software Foundation.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+import {SortFields} from '@axelor/aos-mobile-core';
+
+export const purchase_sortFields: SortFields = {
+  purchase_purchaseRequest: ['statusSelect'],
```

Comment:
sort them by purchaseRequestSeq too to get the latest request as the top for each status